src/libmscore/draw/painter.cpp

```diff
@@ -20,274 +20,234 @@
 
 using namespace mu::draw;
 
-Painter::Painter(QPainter* painter)
-    : m_painter(painter)
+Painter::Painter(IPaintProviderPtr painter)
+    : m_p(painter)
```

Comment:
I think it is better to rename this member to m_provider

Reply:
renamed